### ISSUE TITLE
[SD3 branch] AdamW cannot work properly.

### ISSUE THREAD

**Issue by mliand** [kohya-ss/sd-scripts#2127]
The adamw optimizer caused the nan issue. I tried to use the default Settings on sd3/3.5 and perform full fine-tuning with the adamw optimizer. In the second step, the loss became nan, and the training with adafactor proceeded normally，there must be something wrong somewhere. I trained with bf16 mixed precision.
